zeebe/dynamic-config/src/main/java/io/camunda/zeebe/dynamic/config/state/ExportersConfig.java

```diff
@@ -42,6 +43,23 @@ public ExportersConfig disableExporter(final String exporterName) {
             Optional.empty()));
   }
 
+  public ExportersConfig disableExporters(final Collection<String> exporterNames) {
+    final var builder = ImmutableMap.<String, ExporterState>builder().putAll(exporters);
```

Comment:
:thought_balloon: Weird question maybe, but why use `ImmutableMap` from Guava and not just the ones from the standard library? e.g. `Collections.unmodifiableMap`? :shrug:

Reply:
`ImmutableMap` provides a builder through which we I found very useful when updating from an existing state.